Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCell.swift

```diff
@@ -36,15 +36,33 @@ final class SavedRecipesTableViewCell: UITableViewCell {
         configureLayout()
     }
 
+    override func layoutSubviews() {
+        super.layoutSubviews()
+
+        contentView.frame = contentView.frame.inset(by: UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10))
```

Comment:
Let's add this to `awakeFromNib`

Reply:
![Simulator Screen Shot - iPhone 14 Pro - 2023-07-06 at 02 47 33](https://github.com/motoon-eg/healthy/assets/45472327/0521530d-8bf5-4e9b-978a-c9a3f8e30273)
if putting it in awakeFromNib instead of layoutSubviews